tmail-backend/tmail-third-party/ai-bot/README.md

```diff
@@ -22,4 +22,35 @@ Sample `mailetcontainer.xml` configuration:
         <model>gpt-4o-mini</model>
     </mailet>
 </processor>
-```
```

Comment:
What is included in the `.properties` file, shouldn't we remove it from the mailet configuration?

Reply:
I think @chibenwa here asked to adapt the mailet example regarding the switch to configuration file, not removing the example all together